assets/css/styles.css

```diff
@@ -1 +1,15 @@
+.text-center{
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin:10px auto;
+font-size:30px;
 
+
+}
+p{
```

Comment:
Remove the margins for paragraph bcs it's fixing from this issue #19 @joe-sida

Reply:
okay